packages/backend/src/controllers/fanart.ts

```diff
@@ -0,0 +1,100 @@
+import { FanartDoc } from '@model/fanart'
+import {
+  deleteFanart,
+  getAllFanart,
+  storeFanart,
+  updateFanart,
+} from '@service/fanart'
+import {
+  responseBadRequest,
+  responseCreated,
+  responseInternalServerError,
+  responseSuccess,
+} from '@util/response'
+import { isBoolean, isString, isUndefined, isValidId } from '@util/validate'
+import { Request, Response } from 'express'
+import { FanartQuery } from 'interface/request'
+
+export const getAllFanartController = async (req: Request, res: Response) => {
+  try {
+    //  Gets [limit] fanart after _id [lastId]
+    const { lastId = 'NULL', limit = '10' }: FanartQuery = req.query
+    const fanarts: FanartDoc[] = await getAllFanart(lastId, +limit)
```

Comment:
Could `FanartQuery` not be directly given to `getAllFanart`? It feels like the usage `FanartQuery` in its current state is a bit redundant.

Reply:
This also applies to the `message` counterpart

Reply:
(this is optional since this could be included in a separate refactoring PR)

Reply:
Sorry, I don't get it, what do you mean by a bit redundant?

Reply:
@azuraga the reason he's destructoring the query is because it has more parameters than just lastId and limit, and also to give it default values. Nothing redundant about that as far as I can see.

Reply:
Ah I misread on that part, sorry